src/sync/sync_config.hpp

```diff
@@ -100,6 +104,7 @@ struct SyncConfig {
     SyncSessionStopPolicy stop_policy;
     std::function<SyncBindSessionHandler> bind_session_handler;
     std::function<SyncSessionErrorHandler> error_handler;
+    std::shared_ptr<ChangesetTransformer> transformer;
```

Comment:
This object is owned externally to SyncConfig - an instance of this object is passed in and reused whenever the adapter opens a synced Realm which needs to export changeset information. As is this object is passed into the global notifier on creation - I guess we could instead pass a method which will create this object on demand, but since this object has no state is seems unnecessary.